gix-prompt/tests/prompt.rs

```diff
@@ -65,7 +65,7 @@ mod ask {
     }
 
     #[test]
-    #[cfg(not(unix))]
+    #[cfg(not(any(target_os = "linux", target_os = "freebsd", target_os = "macos")))]
     #[ignore]
     fn username_password_not_available() {}
 }
```

Comment:
Oh, since it's ignored I have a feeling this might be a leftover - something that was supposed to be implemented, but I forgot to mark it with `ignore = "TBD"`. Otherwise it makes no sense and also I don't have context on it anymore :/.

Reply:
This empty ignored test was added in dc2765f (#496), which also added `#[cfg(unix)]` to the `username_password` test. Maybe the commit message "restrict tests to unix only" suggests that this was expressing a limitation of the tests, rather than one believed to apply to the code under test.

Reply:
Sounds good. I've opened #1647 to remove it.